src/lib/components/modal/index.stories.mdx

```diff
@@ -26,6 +26,7 @@ Modals are dialog overlays that prevent the user from interacting with the rest
 | onClose   | `function () => void` | Callback when the user close the modal                                                 | n/a           | true     |
 | children  | React.jsx             | The content that gets displayed on the modal                                           | n/a           | true     |
 | className | string                | Any additional className                                                               | n/a           | false    |
+| required  | boolean               | Prevent the user from closing the modal                                                | false         | false    |
```

Comment:
I'm not sure about the name, might be a bit confusing… what do you think about `dismissable`?

Reply:
Yeah, sounds better. But it would be inversed right?
The default value should be `true` and if set to `false` the user will not be able to close the modal.

Reply:
Indeed!